### DISCUSSION TITLE
VirusTotal detected an engine in the latest Window installer

### DISCUSSION THREAD

**Discussion by watermelonoof** [ImageMagick/ImageMagick#2798] [Help] [answered]
First of all sorry for my terrible English.
https://www.virustotal.com/gui/file/0720ee0c857cddbafa8d19492635274276f5a317ffd36b7c5745cd66c936105f/detection

Gridinsoft accuses the engine `Ransom.Win32.Wacatac.oa!s1` in the Window installer file. What could it be?

**Comment by urban-warrior** [comment, marked as answer]
It could be a false positive.  It won't be the first time. @dlemstra can you confirm.

**Reply by watermelonoof** [reply]
Thank you for looking at this.

**Reply by dlemstra** [reply]
This is indeed a false positive

**Reply by watermelonoof** [reply]
Thanks for clarifying :)
